### PR TITLE
Feat: Update dependencies

### PR DESCRIPTION
**Pull request by tuul-wq**
- Updated by hand a few dependencies that was mentioned by `dependabot`

### CODE REVIEW

File: src/main/main.ts [hunk 1]
```diff
@@ -56,9 +55,10 @@ export async function MainWindow() {
   ENVIRONMENT.IS_DEV && window.webContents.openDevTools({ mode: 'bottom' });
 
   // Open urls in the user's browser
-  window.webContents.on('new-window', (event, url) => {
-    event.preventDefault();
-    shell.openExternal(url);
+  window.webContents.setWindowOpenHandler((details) => {
```
Comment (tuul-wq) [inline src/main/main.ts:58]: Change to align with `Electron v27`

File: src/main/main.ts [hunk 2]
```diff
@@ -78,22 +78,20 @@ export async function MainWindow() {
   return window;
 }
 
-function rotateLogs(oldLogFile: LogFile): void {
+function rotateLogs(oldLogFile: LogFile) {
```
Comment (tuul-wq) [inline src/main/main.ts:81]: Code style changes

File: src/main/factories/create.ts [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import { BrowserWindow, BrowserWindowConstructorOptions, session } from 'electron';
 
-import { APP_CONFIG } from '@/app.config';
+import { APP_CONFIG } from '../../../app.config';
```
Comment (sokolova-an) [inline src/main/factories/create.ts:3]: why change it from `@/app.config` ? Let's have it as it was

File: src/main/main.ts [hunk 4]
```diff
@@ -1,11 +1,10 @@
...
+import { join, parse } from 'path';
+import { readdirSync, rmSync, renameSync } from 'fs';
+import { BrowserWindow, Menu, shell } from 'electron';
 import log, { LogFile } from 'electron-log';
 import windowStateKeeper from 'electron-window-state';
-import * as path from 'path';
-import * as fs from 'fs';
 
-import { APP_CONFIG } from '@/app.config';
+import { APP_CONFIG } from '../../app.config';
```
Comment (sokolova-an) [inline src/main/main.ts:7]: and here

**Review by sokolova-an — APPROVED** [review]

**Review by Asmadek — APPROVED** [review]

File: src/main/factories/create.ts (hunk 3 above)
Comment (tuul-wq) [reply src/main/factories/create.ts:3]: This alias was not working for electron
Requires research

**Pull request merged**